models/Asset/Image/Thumbnail/Config/Dao.php

```diff
@@ -69,7 +71,7 @@ public function exists(string $name): bool
     /**
      * @throws \Exception
      */
-    public function save()
+    public function save($autoClearThumbnails = true)
```

Comment:
I'm just thinking if `$autoClearThumbnails` is really a good idea, because it conflicts in a way with the Symfony config `auto_clear_temp_files`. So even if you call `save(true)` but `auto_clear_temp_files` is set to `false` it won't run the cleanup. 
So I'd really go for a `$forceClearTempFiles = false`, which does call `$this->model->clearTempFiles();` synchronously if set to `true`.

Reply:
You are right. Changed the code as you requested.